client/config/default.json

```diff
@@ -1,6 +1,6 @@
 {
     "axios": {
-        "baseURL": "/api",
+        "baseURL": "https://stg.meirim.org/api",
```

Comment:
why was it changed? We use it for BE in both prod and stg. Would we want to test it before hand in production?

Reply:
we must have it because we now have several envs